mypy.ini

```diff
@@ -20,7 +20,8 @@ force_union_syntax = true
 
 
 plugins =
-    mypy_django_plugin.main
+    mypy_django_plugin.main,
+    mypy.plugins.proper_plugin
```

Comment:
Thanks! I 100% forgot about it 😆 
But, using it in django-stubs was my motivation for adding it to the `mypy.` namespace!

Reply:
Haha, yeah I happened to come across the documentation and started checking it out